.github/workflows/rust.yml

```diff
@@ -76,6 +76,8 @@ jobs:
 
   wasm-test:
     runs-on: ubuntu-latest
+    env:
+      WASM_BINDGEN_TEST_TIMEOUT: "3600" # if our test takes over an hour, it probably _should_ fail
```

Comment:
I'm not sure what the default is, but 1h sounds way too much to keep the runner busy. How about 20-30min?

Reply:
Default is 20s. 20 minutes works for me though.